wurst/objects/items/BootsDefinition.wurst

```diff
@@ -0,0 +1,244 @@
+package BootsDefinition
+
+// Standard-library imports:
+import Assets
+import ObjectIds
+import AbilityObjEditing
+
+// Local imports:
+import Items
+import ColorUtils
+import LocalAssets
+import ToolTipsUtils
+import LocalObjectIDs
+import StringExtensions
+import LocalItemObjEditing
+
+let EVASION_CHANCE = 0.10
+
+let TT_COMMON = "Increases wearer's movement speed by {0}, intelligence by {1}"
+let TT_TRANSMUTE_SUFFIX = "|nAll basic boots transmute into the same bone/iron/steel boots.".color(COLOR_ORANGE)
+
+let TT_UNFORMAT_SKIN_BOOT = TT_COMMON + " and provides warmth."
+let TT_UNFORMAT_ARMOR_BOOT = TT_COMMON + ", armor by {2} and provides warmth."
+let TT_UNFORMAT_ANA_BOOT_STRING = "Increases wearer's movement speed by {0}, armor by {1} and all stats by {2}. Also gives {3} chance to dodge an attack."
+let TT_UNFORMAT_HYDRA_FINS_STRING = "Increases wearer's movement speed by {0}, armor by {1} armor and all stats by {2}. Also gives spell shield with {3} second cooldown."
+
+// We can retrieve the ability data field, returning an Int in those case
+// DataA1 being the bonus movement speed for bonus move speed ability
+// DataB1 being the intelligence bonus field for the all stat bonus ability
+public let TT_ELK_SKIN_BOOTS = TT_UNFORMAT_SKIN_BOOT
```

Comment:
Can you do

    public let TT_ELK_SKIN_BOOTS = TT_UNFORMAT_SKIN_BOOT.format(
        getAbilityDataField(ABILITY_SPEED_BONUS_60,  "DataA1").color(COLOR_ORANGE),
        getAbilityDataField(ABILITY_INT_BONUS_2,     "DataB1").color(COLOR_TURQUOISE)
    )